src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url("./images/star.svg") no-repeat;
+  margin-left: 4px;
+  background-position: 50%;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 > .stars__star:nth-child(-n + 1),
+.stars--2 > .stars__star:nth-child(-n + 2),
+.stars--3 > .stars__star:nth-child(-n + 3),
+.stars--4 > .stars__star:nth-child(-n + 4),
+.stars--5 > .stars__star:nth-child(-n + 5) {
+  background: url("./images/star-active.svg") no-repeat;
+  background-position: 50%;

```

Comment:
You added background-position by default 50% on stars__star class. So you can delete this line. Smaller code - easier read.